apps/src/applab/designMode.js

```diff
@@ -135,6 +135,11 @@ designMode.onPropertyChange = function(element, name, value) {
     case 'width':
       element.style.width = value + 'px';
       element.parentNode.style.width = value + 'px';
+
+      if (element.style.backgroundSize) {
```

Comment:
It absolutely would. I was initially confused by your question, since I thought I had done this already. Clearly, I hadn't.